packages/fetchai/skills/tac_negotiation/handlers.py

```diff
@@ -293,7 +293,9 @@ def _on_accept(self, accept: FipaMessage, fipa_dialogue: FipaDialogue) -> None:
                     contract_address=strategy.contract_address,
                     callable="get_hash_batch",
                     kwargs=ContractApiMessage.Kwargs(
-                        strategy.kwargs_from_terms(fipa_dialogue.terms)
+                        strategy.kwargs_from_terms(
+                            fipa_dialogue.terms, is_from_terms_sender=False
+                        )
```

Comment:
Another problem was how the terms are translated into kwargs for the contract api message...